tests/dummy/app/components/flatpickr-wrapper.hbs

```diff
@@ -0,0 +1,17 @@
+{{! BEGIN-SNIPPET flatpickr-wrapper.hbs }}
+<@labelComponent />
+
+<div class="{{if @isValid 'is-valid'}} {{if @isInvalid 'is-invalid'}}">
+  <EmberFlatpickr
+    placeholder={{@placeholder}}
+    @date={{@value}}
+    @onChange={{@update}}
+    class="form-control"
+    ...attributes
+  />
+
+</div>
+
+<@hintComponent />
+<@errorComponent class="d-block" />
```

Comment:
What's that class?

Reply:
Im not sure :sweat_smile: a copied it from the `favorite-colors` component but there seems to be no selector matching it. I removed it in both components.

Reply:
Got it: https://getbootstrap.com/docs/4.0/utilities/display/